frontend/src/components/Home.tsx

```diff
@@ -55,16 +57,34 @@ function Home(props:Props) {
     }
 
     return (
-        <div>
-            <h1>Login & Registration</h1>
-            <p>Please fill out the form below to register for an account!</p>
-            <form onSubmit={onSubmit}>
-                <input type="email" placeholder="Email" required onChange={onChangeHandlerUsername}/>
-                <input type="password" placeholder="Password" required minLength={3} onChange={onChangeHandlerPassword} />
-                <button type="submit" className="login" onClick={onClickLogin}>Login</button>
-                <button type="submit" className="register" onClick={onClickRegister}>Register</button>
-            </form>
-        </div>
+        <Container>
+            <Row>
+                <h1>Login & Registration</h1>
+            </Row>
+            <Row>
+                <p>Please fill out the form below to register for an account!</p>
+            </Row>
+            <Form onSubmit={onSubmit}>
```

Comment:
seems like everything looks fine with the form. for future projects i can recommend the formik library. it makes the handling easier and has nice validation possibilities

Reply:
see above